install_script.sh.template

```diff
@@ -1053,6 +1053,20 @@ If the cause is unclear, please contact Datadog support.
     fi
     if [ -n "$install_updater" ]; then
       packages=("datadog-updater" "datadog-signing-keys")
+      services=()
+      config_file="/etc/datadog-agent/datadog.yaml"
+      if [ ! -e $config_file ]; then
+        $sudo_cmd mkdir -p /etc/datadog-agent
+        $sudo_cmd touch $config_file
+        $sudo_cmd chmod 644 $config_file
+        $sudo_cmd sh -c "echo 'api_key: $apikey' > $config_file"
+        $sudo_cmd sh -c "echo 'site: $site' >> $config_file"
+        $sudo_cmd sh -c "echo 'hostname: $hostname' >> $config_file"
+        formatted_host_tags="['""$( echo "$host_tags" | sed "s/,/', '/g" )""']"  # format `env:prod,foo:bar` to yaml-compliant `['env:prod','foo:bar']`
+        $sudo_cmd sh -c "echo 'tags: $formatted_host_tags' >> $config_file"
+        $sudo_cmd sh -c "echo 'env: $env' >> $config_file"
+        $sudo_cmd sh -c "echo 'dd_url: $url' >> $config_file"
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ Are all of these required for the updater? We discussed some in our standup and it was pointed out most of the configuration would be done from our UI - using remote config, right?

Reply:
Yes all of them would be useful.
- apiKey is used for auth
- env + host tags + hostnames are used for configuration routing, we will match on these to know fetch the installation configration
- dd_url/site are used to target the correct datadog